### PR TITLE
Tickets 14822 14808

### PR DESCRIPTION
**Pull request by lewisrenfrew**

### CODE REVIEW

**Review by andrewfraser73 — APPROVED** [review]
looks ok

File: src/Domain.LinnApps/Parts/MechPartSourceService.cs [hunk 1]
```diff
@@ -63,14 +58,161 @@ public string GetCapacitanceLetterAndNumeralCode(string unit, decimal value)
...
+            current.QualityVerifiedBy = updated.QualityVerifiedBy;
+            current.QualityVerifiedDate = updated.QualityVerifiedDate;
+            current.RemoveTCodeBy = updated.RemoveTCodeBy;
+            current.RemoveTCodeDate = updated.RemoveTCodeDate;
+            current.VerifiedBy = updated.VerifiedBy;
+            current.VerifiedDate = updated.VerifiedDate;
+            current.PurchasingQuotes = updated.PurchasingQuotes;
+            current.Usages = updated.Usages;
+            current.LifeExpectancyPart = updated.LifeExpectancyPart;
+            current.Configuration = updated.Configuration;
```
Comment (iaincrawf) [inline src/Domain.LinnApps/Parts/MechPartSourceService.cs:184]: They might already be somewhere but are there tests for these million bits of the MechPartSources to check they update the right thing? Looks fine to my eye but there's just so many

**Review by iaincrawf — APPROVED** [review]
I don't really understand any context but code looks good to me anyway 😄

File: src/Domain.LinnApps/Parts/MechPartSourceService.cs (hunk 1 above)
Comment (lewisrenfrew) [reply src/Domain.LinnApps/Parts/MechPartSourceService.cs:184]: No.. there are tests for anything thats more complicated than just a this.that = that.that but you'r e right, I should add tests for all of this incredibly boring code

**Pull request merged**